client/packages/coldchain/src/Equipment/ImportAsset/UploadTab.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { FC, useState } from 'react';
+import Papa from 'papaparse';
+import { ImportPanel } from './ImportPanel';
+import { useNotification } from '@common/hooks';
+import { InlineProgress, Typography, Upload } from '@common/components';
+import { useTranslation } from '@common/intl';
+import {
+  Grid,
+  Stack,
+  Paper,
+  Link,
+  FnUtils,
+  FileUtils,
+  InsertAssetInput,
+} from '@openmsupply-client/common';
+import * as EquipmentImportModal from './EquipmentImportModal';
+import { ImportRow, toInsertEquipmentInput } from './EquipmentImportModal';
+import { importEquipmentToCsv } from '../utils';
+import { AssetCatalogueItemFragment } from '@openmsupply-client/system';
+interface EquipmentUploadTabProps {
+  setEquipment: React.Dispatch<React.SetStateAction<ImportRow[]>>;
+  setErrorMessage: (value: React.SetStateAction<string>) => void;
+  onUploadComplete: () => void;
+  catalogueItemData?: AssetCatalogueItemFragment[];
+}
+
+export const EquipmentUploadTab: FC<ImportPanel & EquipmentUploadTabProps> = ({
+  tab,
+  setErrorMessage,
+  setEquipment,
+  onUploadComplete,
+  catalogueItemData,
+}) => {
+  const t = useTranslation('coldchain');
+  const { error } = useNotification();
+  const [isLoading, setIsLoading] = useState(false);
+  const EquipmentBuffer: EquipmentImportModal.ImportRow[] = [];
+
+  const csvExample = async () => {
+    const emptyRows: ImportRow[] = [];
+    const csv = importEquipmentToCsv(
+      emptyRows.map((row: ImportRow): InsertAssetInput => {
+        return toInsertEquipmentInput(row, catalogueItemData);
+      }),
+      t
+    );
+    FileUtils.exportCSV(csv, t('filename.cce'));
+  };
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const csvImport = (files: any) => {
```

Comment:
No need for the `any`
```suggestion
const csvImport = <T extends File>(files: T[]) => {
```

just change this line also:

```
    if (!csvFile?.name.endsWith('.csv')) {
```